app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -1239,7 +1242,14 @@ public void seekTo(final long positionMillis) {
             Log.d(TAG, "seekBy() called with: position = [" + positionMillis + "]");
         }
         if (simpleExoPlayer != null) {
-            simpleExoPlayer.seekTo(positionMillis);
+            long normalizedPositionMillis = positionMillis;
+            if (normalizedPositionMillis < 0) {
+                normalizedPositionMillis = 0;
+            } else if (normalizedPositionMillis > simpleExoPlayer.getDuration()) {
+                normalizedPositionMillis = simpleExoPlayer.getDuration();
+            }
+
+            simpleExoPlayer.seekTo(normalizedPositionMillis);
```

Comment:
Is this done to prevent negative positions when using rewind? If this is the case, please add a comment in the code to clarify why the check is necessary.